snuba/admin/views.py

```diff
@@ -0,0 +1,49 @@
+from typing import Dict, Text, Tuple
+
+import simplejson as json
+from flask import Flask, Response, jsonify, make_response, request
+from prettytable import PrettyTable
+
+from snuba import state
+from snuba.admin.clickhouse.system_queries import SystemQuery, run_query
+
+application = Flask(__name__, static_url_path="", static_folder="dist")
+
+
+@application.route("/")
+def root() -> Response:
+    return application.send_static_file("index.html")
+
+
+@application.route("/clickhouse_queries")
+def clickhouse_queries() -> Response:
+    res = [q.to_json() for q in SystemQuery.all_queries()]
+    return make_response(jsonify(res), 200)
+
+
+@application.route("/run_clickhouse_query", methods=["POST"])
+def clickhouse() -> str:
+    # TODO: You can do something like this to get all the hosts:
+    # SELECT * FROM system.clusters
+    req = request.get_json()
+    print("REQUEST: ", request, req)
+    results, columns = run_query(
+        req.get("host", "localhost"),
+        req.get("storage", "transactions"),
+        req.get("query_name"),
```

Comment:
I think you should be already able to use the Clusters here. If you open the clusters from the snuba settings, a cluster can give you the list of hosts.
https://github.com/getsentry/snuba/blob/master/snuba/clusters/cluster.py#L336